src/home/home.html

```diff
@@ -28,7 +28,7 @@
     </div>
     <div class="section launches">
       <template if.to-view="featuredSeeds.length">
-        <div class="title heading heading1">Featured Launches</div>
+        <div class="title heading heading3">Featured Launches</div>
```

Comment:
please just override the css for heading1 rather than creating in home.scss a heading3 (which also exists in styles.scss)

Reply:
I thought the &.heading3 makes sure to only style any heading3 that is a child of the parent element

Reply:
I thought heading3 might be better because headin1 has a gradient as it background that might require more CSS to change it style and heading3 only needs to change the font-size